builder/main.py

```
@@ -1908,6 +1908,170 @@ def download_fs_action(target, source, env):
+
+    # CLI arguments after --
+    if "--" in sys.argv:
+        dash_index = sys.argv.index("--")
+        if dash_index + 1 < len(sys.argv):
+            base_cmd.extend(sys.argv[dash_index + 1:])
+
+    # Process files in batches to avoid OS command-line length limits
+    try:
+        for i in range(0, len(source_files), _CLANG_FORMAT_BATCH_SIZE):
+            batch = source_files[i:i + _CLANG_FORMAT_BATCH_SIZE]
+            result = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)